demo/fallback.html

```diff
@@ -0,0 +1,152 @@
+<html>
+
+<head>
+  <title>Fallback element</title>
+  <meta name="viewport" content="width=device-width, minimum-scale=1.0, initial-scale=1, user-scalable=yes">
+
+  <style>
+    body {
+      font-family: Roboto, Helvetica, sans-serif;
+      max-width: 600px;
+      margin: 10px auto;
+    }
+
+    virtual-list {
+      width: 100%;
+      height: 50vh;
+      margin: 10px 0;
+      border-top: 2px solid #eee;
+      border-bottom: 2px solid #eee;
+    }
+
+    fallback-element {
+      display: block;
+      width: 100%;
+      text-align: center;
+      line-height: 80px;
+      border-bottom: 1px solid #eee;
+      animation: inverse-color 1000ms infinite;
+    }
+
+    @keyframes inverse-color {
+      50% {
+        background-color: #ddd;
+        color: white;
+      }
+    }
+  </style>
+</head>
+
+<body>
+  <p>
+    <label>
+      Server response time (ms):
+      <input type="number" value=1000 min=0 max=10000 onchange="updateDelay(this)">
+    </label>
+  </p>
+
+  <virtual-list></virtual-list>
+
+  <script type="module">
+    import '../virtual-list-element.js';
+    import './contacts/contact-element.js';
+
+    const fallbackHeight = 80;
+    const fallbacksCount = Math.ceil(innerHeight / fallbackHeight) + 5;
```

Comment:
Is this window.innerHeight? Maybe being explicit is better?

A comment explaining why this is calculated this way would be nice.